src/backend/main.ts

```diff
@@ -1339,6 +1337,12 @@ ipcMain.handle(
         status: 'syncing-saves'
       })
 
+      sendFrontendMessage('gameStatusUpdate', {
```

Comment:
This is duplicated from above

Reply:
It is not actually.
It sets to `done`, so the GlobalState knows that the game session stopped.
Then it changes the status to `syncing-saves`. Then It sets to `done` again.
So, `done` is not only the state for stopping playing but for finishing any other process that came before.
So this is correct.

Reply:
Could you add a comment explaining this here?

Reply:
added the comment